src/geophires_x/CylindricalReservoir.py

```diff
@@ -244,17 +251,19 @@ def Calculate(self, model: Model) -> None:
                                                  ) / 1e15  # 10^15 J
         self.cpwater.value = heat_capacity_water_J_per_kg_per_K(
             model.wellbores.Tinj.value * 0.5 + (self.Trock.value * 0.9 + model.wellbores.Tinj.value * 0.1) * 0.5,
-            pressure=self.hydrostatic_pressure()
+            pressure=model.reserv.lithostatic_pressure()
         )
         self.rhowater.value = density_water_kg_per_m3(
             model.wellbores.Tinj.value * 0.5 + (self.Trock.value * 0.9 + model.wellbores.Tinj.value * 0.1) * 0.5,
-            pressure=self.hydrostatic_pressure()
+            pressure=model.reserv.lithostatic_pressure()
```

Comment:
We just changed these from lithostatic to hydrostatic last month (https://github.com/NREL/GEOPHIRES-X/pull/217 / https://github.com/NREL/GEOPHIRES-X/commit/c77f4183b445f1e1767dd3d62fda68436c7f7acd), is this change back intentional? If so, what's the rationale?

Reply:
I had merge conflicts during the class which I had to resolve quickly. We should go back to hydrostatic.